common/pom.xml

```diff
@@ -74,9 +74,13 @@
       <artifactId>jdbm</artifactId>
     </dependency>
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
     </dependency>
+      <dependency>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-1.2-api</artifactId>
```

Comment:
Note: This library is still required - it offers the log4j 1.2 API for core-geonetwork (and maybe some other dependencies to call). The backend is log4j 2.

NOTE: Changing the core-geonetwork to use the log4j 2 api directly can be done as part of this pull-request or as separate pull request (which I recommend) after this one is merged.